docs/using/python.md

```diff
@@ -10,164 +10,161 @@ has_children: false
 # Calling the lakeFS API from Python
 {: .no_toc }
 
-The [lakeFS API](../reference/api.md){: target="_blank" } is OpenAPI 2.0 compliant, allowing the dynamic generation of clients from multiple languages.
+The [lakeFS API](../reference/api.md){: target="_blank" } is OpenAPI 3.0 compliant, allowing the generation of clients from multiple languages.
 
-For Python, this example uses [Bravado](https://github.com/Yelp/bravado){: target="_blank" }
-which generates a dynamic client at runtime, from an OpenAPI definition served by a lakeFS server.
+For Python, this example uses [lakeFS's python package](https://pypi.org/project/lakefs-client/){: target="_blank" }.
+The lakefs-client pacakge was created by [OpenAPI Generator](https://openapi-generator.tech){: target="_blank" } using our OpenAPI definition served by a lakeFS server.
```

Comment:
I would add a note that this is the recommended client package for python, but other solutions are valid too (Bravado, simple HTTP client)

Reply:
I'll add a note that lakeFS can be accessed using any OpenAPI v3 library or directly by HTTP client.
Bravado can't be used - supports only OpenAPI v1 and v2.